plugins/modules/ome_application_certificate.py

```diff
@@ -209,45 +243,61 @@ def format_csr_string(csr_string):
         "department_name": {"required": False, "type": "str"},
         "business_name": {"required": False, "type": "str"},
         "locality": {"required": False, "type": "str"},
         "country_state": {"required": False, "type": "str"},
         "country": {"required": False, "type": "str"},
         "email": {"required": False, "type": "str"},
-        "upload_file": {"required": False, "type": "str"},
-        "subject_alternative_names": {"required": False, "type": "str"}
+        "subject_alternative_names": {"required": False, "type": "str"},
+        "upload_file": {"required": False, "type": "str"}
     }
 
     module = OmeAnsibleModule(
         argument_spec=specs,
         required_if=[["command", "generate_csr", ["distinguished_name", "department_name",
                                                   "business_name", "locality", "country_state",
                                                   "country", "email"]],
-                     ["command", "upload", ["upload_file"]]],
+                     ["command", "upload", ["upload_file"]],
+                     ["command", "upload_cert_chain", ["upload_file"]]],
         supports_check_mode=False
     )
+
     header = {"Content-Type": "application/octet-stream", "Accept": "application/octet-stream"}
     try:
         with RestOME(module.params, req_session=False) as rest_obj:
-            method, uri, payload = get_resource_parameters(module)
             command = module.params.get("command")
-            dump = False if command == "upload" else True
-            headers = header if command == "upload" else None
+            if command == "upload_cert_chain":
+                ome_version = get_ome_version(rest_obj)
+                if LooseVersion(ome_version) < "3.10":
+                    module.exit_json(msg=LOW_OME, skipped=True)
+
+            method, uri, payload = get_resource_parameters(module)
+            dump = False if command in ["upload", "upload_cert_chain"] else True
+            headers = header if command in ["upload", "upload_cert_chain"] else None
             resp = rest_obj.invoke_request(method, uri, headers=headers, data=payload, dump=dump)
             if resp.success:
                 if command == "generate_csr":
                     resp_copy = resp.json_data
-                    formated_csr = format_csr_string(resp_copy["CertificateData"])
-                    resp_copy["CertificateData"] = formated_csr
+                    formatted_csr = format_csr_string(resp_copy["CertificateData"])
+                    resp_copy["CertificateData"] = formatted_csr
                     module.exit_json(msg="Successfully generated certificate signing request.",
                                      csr_status=resp_copy)
-                module.exit_json(msg="Successfully uploaded application certificate.", changed=True)
+                else:
+                    module.exit_json(msg="Successfully uploaded application certificate.", changed=True)
```

Comment:
Move the success and error messages to constant. 
Please change wherever applicable

Reply:
Done